app/etl.py

```diff
@@ -661,18 +681,39 @@ def _compute_kraj_vaccinated(self):
 
     def _compute_kraj_distributed(self):
         """Computes metrics based on distributed vaccines dataset for each kraj."""
-        distributed = db.session.query(
-            Kraj.id, func.sum(OckovaciMistoMetriky.vakciny_prijate_pocet).label('vakciny_prijate_pocet')
-        ).join(Okres, Okres.kraj_id == Kraj.id) \
-            .join(OckovaciMisto, (OckovaciMisto.okres_id == Okres.id)) \
-            .join(OckovaciMistoMetriky, OckovaciMistoMetriky.misto_id == OckovaciMisto.id) \
-            .filter(OckovaciMistoMetriky.datum == self._date) \
-            .group_by(Kraj.id) \
-            .all()
+        distributed = db.session.query("kraj_id", "vakciny_prijate_pocet").from_statement(text(
```

Comment:
I have doubts about the performance of this query. Definitely 1st and 3rd subselect can we used in with clause. But if this is fast, no issue.

Reply:
performance is not the best, but it's suitable, I will try to improve it in the following PR